classes/sdimg.bbclass

```diff
@@ -39,21 +42,24 @@ IMAGE_BOOT_ENV_FILE ?= "uboot.env"
 # u-boot environment file
 IMAGE_UENV_TXT_FILE ?= "uEnv.txt"
 
+# These are usually defined in the machine section, but for daisy this
+# concept doesn't exist yet.
+IMAGE_BOOT_FILES ?= "u-boot.${UBOOT_SUFFIX}"
+IMAGE_BOOT_FILES_append_beaglebone ?= " MLO"
```

Comment:
Hmm, my impression is that it's better to leave things so they can be overridden by users of the layer. I'm unsure what's the best practice here. @bboozzoo: Do you have any advice?